content/scalers/apache-kafka-topic.md

```diff
@@ -33,9 +33,11 @@ This specification describes the `kafka` trigger for Apache Kafka Topic.
 
  You can use `TriggerAuthentication` CRD to configure the authenticate by providing authMode, username, password. If your kafka cluster does not have sasl authentication turned on, you will not need to pay attention to it.
 
+ The sasl_ssl_plain authMode can be used when using the Kafka head on Azure Event Hubs to act as trigger.
```

Comment:
Same as above, who uses AEH knows that it supports SASL_SSL_PLAIN so the trigger has to be configured in that way.

Reply:
I think this is a fair point in the documentation, what is your concern with this? We provide several means to authenticate and here we just point out that this works with Event Hubs, don't see an issue with that.

Reply:
@ppatierno, actually anyone who has played with producers and consumers to AEH w/Kafka endpoint would know that this is the way to authenticate with it. I feel the mention here does more good than harm :)

Reply:
I see it the other way around. :-) The point I am making here is that the user who uses EH and is interested in using it via Kafka's head should read the Microsoft official doc to check what kind of authentication it supports. At that point, if he wants to use KEDA then he should already know to set `sasl_ssl_plain` as auth mode so it's not worth mentioning EH here.

Reply:
I've modified the doc to read `The sasl_ssl_plain authMode can also be used when using the Kafka head on Azure Event Hubs to act as trigger.`

Reply:
Which again should be something that people using Azure EH with Kafka should already know :-)
As the same point I tried to make before: imagine that the project Apache Pulsar will add a Kafka protocol head, at this point we should update the doc saying " ... when using the Kafka head on Azure Event Hubs or Apache Pulsar to act as trigger". It should happen again and again for other new services with same support. I would be more generic not mentioning specific cases but leaving them as dedicated examples.

Reply:
Have made the requested changes